specs/sb/api.md

```diff
@@ -296,6 +296,64 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
   <td>boolean</td>
   <td>This field allows the plan to be limited by the non_basic_services_allowed field in a Cloud Foundry Quota, see <a href="http://docs.cloudfoundry.org/running/managing-cf/quota-plans.md">Quota Plans</a>. Default: true</td>
 </tr>
+<tr>
+  <td>&nbsp;&nbsp;&nbsp;schemas</td>
+  <td>object</td>
+  <td>The schema definitions for the input and output properties for service instances and bindings of the plan.</td>
+</tr>
+</tbody>
+</table>
+
+<h5> Schemas Object <a name="PObject"></a> </h5>
+
+<table border="1" class="nice">
+<thead>
+<tr>
+  <th>Response field</th>
+  <th>Type</th>
+  <th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td>&nbsp;&nbsp;&nbsp;service_instance</td>
+  <td>object</td>
+  <td>The schema definitions for the input and output properties for a service instance.</td>
+</tr>
+<tr>
+  <td>&nbsp;&nbsp;&nbsp;service_binding</td>
+  <td>object</td>
+  <td>The schema definitions for the input and output properties for a service binding. Used only if the service is bindable.</td>
+</tr>
+</tbody>
+</table>
+
+<h5> Schema Object <a name="PObject"></a> </h5>
```

Comment:
See my comments above Doug, I chose to go with consistence within the doc, where sub objects are broken out into their own section, e.g., with the Plan Object. I have however added hyperlinks as was done with the plan field.